### ISSUE TITLE
Regression on #6987 alternate fix for .dot file handling

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#6999]
In the CGAL BGL package we get the error message:
```
error: Problems running dot: exit code=2, command='dot', arguments='"/home/cgal-testsuite/cgal_doc_build/CGAL-5.0-Ic-39/doc/scripts/doc_master/doc_output/BGL/./inline_dotgraph_1.dot.dot" -Tpng -o "/home/cgal-testsuite/cgal_doc_build/CGAL-5.0-Ic-39/doc/scripts/doc_master/doc_output/BGL/dot_inline_dotgraph_1.png"'
error: Problems running dot: exit code=2, command='dot', arguments='"dot_inline_dotgraph_1.dot" -Tcmapx -o "/home/cgal-testsuite/cgal_doc_build/CGAL-5.0-Ic-39/doc/scripts/doc_master/doc_output/BGL/dot_inline_dotgraph_1.map"'
```
The executable used was the one that was created just after the mege of pull request #6987 

The problem here is caused by the `\dot` command but also the `\dotfile` commands gives these errors.

Some problem occurs with the doxygen test  procedure number 22 when running with the `--xhtml` option (same happens with other output formats as well).

I have created from the CGAL problem a small example: [regression.zip](https://github.com/doxygen/doxygen/files/3194050/regression.zip)

**Comment by albert-github** [comment]
Setting issue to "Fixed but not releassed" as the cause of the problem has been reverted.

As written in #7000 this problem has to be considered as well when implementing the solution for #6960 by #6987

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
